lib/elementWrapper/dropDownWrapper.js

```diff
@@ -51,12 +51,12 @@ class DropDownWrapper extends ValueWrapper {
   }
 
   /**
-   * Selects the option corresponding to value passed.
-   * @param {any} value value to be selected
+   * Select option or options from a drop down
+   * @param {string[]} values value(s) to be selected
```

Comment:
Can you also add a few examples usages of using array as parameters? You can add it here using the `@example` tag and at `dropDown` API jsdoc which will then appear at https://docs.taiko.dev/api/dropdown/

Reply:
Added.